Pipfile

```diff
@@ -30,7 +30,6 @@ pytest-flask = "*"
 pytest = "*"
 pytest-sugar = "*"
 responses = "*"
-types-simplejson = "*"
```

Comment:
It was already here, so you didn't need to change anything. You just need to revert Pipfile and pipfile.lock

Reply:
Okay, will do

Reply:
Look like the file is still there. Would need revert to `main`.